pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -602,7 +602,7 @@ def plot_animated_field(
         ----------
         quantity : str
             Quantity to plot (e.g. ``"Mag_E"``).
-        objects : list, str
+        assignment : list, str
             List of objects or faces to apply the field plot to.
```

Comment:
```suggestion
            One or more objects or faces to apply the field plot to.
```
Earlier, str was listed as the first data type, but here list is first. Make them consistent with the singular first (since "One or more") is used in the description?